frontend/lib/identification/identification_view.dart

```diff
@@ -30,21 +35,49 @@ class _IdentificationPageState extends State<IdentificationPage> {
             title: Text('Digitale Ehrenamtskarte'),
           ),
           body: SingleChildScrollView(
-            child: Container(
-              padding: EdgeInsets.symmetric(horizontal: 16, vertical: 16),
-              child: _cardDetails == null
-                  ? NoCardView(
-                      onOpenQrScanner: () => openQRCodeScannerView(context),
-                    )
-                  : CardDetailView(
-                      cardDetails: _cardDetails,
-                      onOpenQrScanner: () => openQRCodeScannerView(context),
+              child: Column(
+            children: [
+              Center(
+                child: RaisedButton(
+                    padding: EdgeInsets.symmetric(vertical: 12, horizontal: 24),
+                    onPressed: () => _showVerificationPage(context),
+                    color: Theme.of(context).primaryColor,
+                    shape: RoundedRectangleBorder(
+                      borderRadius: BorderRadius.circular(32.0),
```

Comment:
button glitches under the appbar at the top. Just a tiny bit but it looks ugly, see screenshot in other comment

Reply:
I also think, the button should probably move to the bottom and it should probably not be blue, as it is not very important for most users.

Reply:
I moved it to the bottom, I'll not change more here as this is not part of this PR.